src/aui-form-builder/js/aui-form-builder-layout-builder.js

```diff
@@ -109,6 +109,16 @@ A.FormBuilderLayoutBuilder.prototype = {
     },
 
     /**
+     * Executed after the `layout-builder:moveEnd` is fired.
+     *
+     * @method _afterLayoutBuilderMoveEnd
+     * @protected
+     */
+    _afterLayoutBuilderMoveEnd: function() {
```

Comment:
I think that we're missing the test for checking that the "add fields" feature is indeed being turned on/off due to a move happening.